packages/fluent-theme/src/fluent/styles/Link.styles.ts

```diff
@@ -0,0 +1,17 @@
+import { ILinkStyleProps } from 'office-ui-fabric-react/lib/Link';
+
+export const LinkStyles = (props: ILinkStyleProps) => {
+  const { isDisabled } = props;
+
+  return {
+    root: [
+      !isDisabled && {
+        selectors: {
+          '&:active, &:hover, &:active:hover': {
+            textDecoration: 'underline'
```

Comment:
@betrue-final-final Should all links have underlines? If I recall right from when we discussed this way back, I thought that underline was a new variant and not necessarily the default.

Reply:
No, just the underline variant. However all links should underline on hover.

Reply:
I read the code wrong, this is just on active and hover. We'll have to track the new underlined variant separate from the Fluent theme project because it requires a change to the component itself.